js/src/features/project/actions.ts

```diff
@@ -105,7 +111,45 @@ export const closeProjectTeamCreateFormAction = createAction(
 )();
 
 export const fetchProjectTeamAsync = createAsyncAction(
-  "FETCH_TEAM_REQUEST",
-  "FETCH_TEAM_SUCCESS",
-  "FETCH_TEAM_FAILED",
-)<string, ProjectTeam, any>();
+  "FETCH_PROJECT_TEAM_REQUEST",
+  "FETCH_PROJECT_TEAM_SUCCESS",
+  "FETCH_PROJECT_TEAM_FAILED",
+)<string, ProjectTeam, FormError[]>();
+
+export const editProjectTeamMemberRoleAsync = createAsyncAction(
+  "EDIT_PROJECT_TEAM_MEMBER_ROLE_REQUEST",
+  "EDIT_PROJECT_TEAM_MEMBER_ROLE_SUCCESS",
+  "EDIT_PROJECT_TEAM_MEMBER_ROLE_FAILED",
+)<EditProjectTeamMemberRoleRequest, undefined, FormError[]>();
+
+export const removeProjectTeamMemberAsync = createAsyncAction(
+  "REMOVE_PROJECT_TEAM_MEMBER_REQUEST",
+  "REMOVE_PROJECT_TEAM_MEMBER_SUCCESS",
+  "REMOVE_PROJECT_TEAM_MEMBER_FAILED",
+)<RemoveProjectTeamMemberRequest, undefined, FormError[]>();
+
+export const addProjectTeamMemberAsync = createAsyncAction(
+  "ADD_PROJECT_TEAM_MEMBER_REQUEST",
+  "ADD_PROJECT_TEAM_MEMBER_SUCCESS",
+  "ADD_PROJECT_TEAM_MEMBER_FAILED",
+)<AddProjectTeamMemberRequest, undefined, FormError[]>();
+
+export const openProjectTeamAddMemberFormAction = createAction(
+  "OPEN_PROJECT_TEAM_ADD_MEMBER_FORM",
+)();
+
+export const closeProjectTeamAddMemberFormAction = createAction(
+  "CLOSE_PROJECT_TEAM_ADD_MEMBER_FORM",
+)();
+
+export const changeProjectTeamStatusAsync = createAsyncAction(
+  "CHANGE_PROJECT_TEAM_STATUS_REQUEST",
+  "CHANGE_PROJECT_TEAM_STATUS_SUCCESS",
+  "CHANGE_PROJECT_TEAM_STATUS_FAILED",
+)<ChangeProjectTeamStatusRequest, undefined, FormError[]>();
+
+export const fetchEmployeesAsync = createAsyncAction(
```

Comment:
Could you reuse already created employee logic in hr module?

Reply:
fix